curriculum/challenges/english/01-responsive-web-design/basic-css-cafe-menu/part-016.md

```diff
@@ -11,10 +11,18 @@ Now that you have the CSS in the `styles.css` file, go ahead and remove the `sty
 
 # --hints--
 
-Test 1
+You should not have any `style` tags in your code.
 
 ```js
+assert(!code.match(/style/i));
+```
+
+You should not have any CSS selectors in your HTML file.
```

Comment:
I think this challenge asks for "rewriting the styles you have created into the styles.css file." and the next challenge will ask for removing the css from html.